tests/build/_internal/test_cmake_file.py

```diff
@@ -40,13 +43,19 @@ def test_returns_rendered_content(self):
         target["macros"] = ["macbaz"]
         target["device_has"] = ["stuff"]
         target["core"] = ["core"]
+        target["c_lib"] = ["c_lib"]
+        target["printf_lib"] = ["printf_lib"]
         target["supported_form_factors"] = ["arduino"]
         config = ConfigFactory()
         toolchain_name = "baz"
+        target["supported_c_libs"] = {toolchain_name: ["small", "std"]}
         result = _render_mbed_config_cmake_template(target, config, toolchain_name, "target_name")
 
         for label in target["labels"] + target["extra_labels"]:
             self.assertIn(label, result)
 
         for macro in target["features"] + target["components"] + [toolchain_name]:
             self.assertIn(macro, result)
+
+        for supported_c_lib in target["supported_c_libs"]:
```

Comment:
Looks like we want to iterate over the `.values()` here, at the moment we're checking the key (which is the toolchain name) is in the rendered template.

Reply:
Is [this](https://github.com/ARMmbed/mbed-tools/compare/b0e0c663fb9e74f4ce5d2e2d1c29e311d77c594f..2797d81a6358ac8c7bd4eb63918f31fdb670c305) ok?

Reply:
Looks ok, but you've force pushed over my commit to pin the `black` version. Please reinstate `black = "==19.10b0"`.

Reply:
I restored it right after. See the current state of that file in https://github.com/ARMmbed/mbed-tools/pull/73/files

Reply:
Sorry. Missed that. LGTM.